deploy/scripts/src/common/utils/custom_metric/counter.ts

```diff
@@ -0,0 +1,4 @@
+import { Counter } from 'k6/metrics'
+
+export const iterationsStarted = new Counter('Iterations_Started')
```

Comment:
Add a check in `unit-tests.ts` for these utilities, to check they are behaving as expected

Reply:
Added a test for custom metric check. We cannot get the value of a counter from inside a k6 test so I have added a threshold check on the counters.